README.md

```diff
@@ -30,7 +30,7 @@
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> This project is built in HTML, CSS, Javascript and some Bootstrap
+> This project is built in HTML, CSS and Javascript
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you to sure you include the author [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) all  the links that can find the author and design :+1: